src/canopy/llm/openai.py

```diff
@@ -30,20 +30,18 @@ def __init__(self,
                  model_name: str = "gpt-3.5-turbo",
                  *,
                  model_params: Optional[ModelParams] = None,
```

Comment:
We probably need to update our `ModelParams` class as well. Please take a look and let's come up with a design. Optional solutions:
- Hard-coded set of params, which we believe that are common to all LLMs. Don't allow other params even if OpenAI has them.
- Hard-coded set of params, but we allow "kwargs" (pydantic extra or similar)
- Free dictionary. Might be problematic if we need to "translate" between external API and underlying models.

Reply:
we chose free dict